### PR TITLE
DDF-3596 Upgraded to Jacoco 0.8.0

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Upgrade to Jacoco 0.8.0
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@paouelle 
@oconnormi 
@peterhuffer 
#### Select relevant component teams: 
@codice/build 
@codice/test 
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@lessarderic 
@clockard 

#### How should this be tested? (List steps with links to updated documentation)
Windows build passes 

#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-3596
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/2945)

### CODE REVIEW

**Comment by shaundmorris** [comment]
build now

File: broker/broker-route-manager/pom.xml [hunk 1]
```diff
@@ -103,7 +103,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.68</minimum>
+                                            <minimum>65%</minimum>
```
Comment (paouelle) [inline broker/broker-route-manager/pom.xml]: ❓ What the reason the expected coverage had to be reduced?

**Comment by clockard** [comment]
I assume the coverage changes are to try out using the % notation.

**Comment by paouelle** [comment]
That part I gathered, my question is about why was it reduced.

File: broker/broker-route-manager/pom.xml (hunk 1 above)
Comment (shaundmorris) [reply broker/broker-route-manager/pom.xml]: Oh yeah, I looked into this more.  We're jumping a few versions (patches, then minor) and nothing stood out. The module only has one small class in it. I could to look to add one but most of the lines left uncovered are catches for exceptions. I would have been more concerned if we had to update several files.

Comment (paouelle) [reply broker/broker-route-manager/pom.xml]: I was not really asking to add more tests although that is always a good idea 😉 
I was just wondering why you had to reduce the coverage from 68% to 65% just by bumping the plugin version. If it is due to the plugin version then a lot of code will be impacted. Hopefully the new version is much better at checking coverage for the Java 8 streams and lambdas.

Comment (shaundmorris) [reply broker/broker-route-manager/pom.xml]: Yes understood. I was surprised only 2 modules needed to updated. They seem to be really small so not sure if that was the big factor. In both cases the module has a single class in it. They also added Java 9 support in this version, but not sure how it affected streams and lambdas.

**Comment by shaundmorris** [comment]
build now

File: broker/broker-route-manager/pom.xml [hunk 2]
```diff
@@ -103,7 +103,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.68</minimum>
```
Comment (lessarderic) [inline broker/broker-route-manager/pom.xml]: ❓Do you think we should keep those as decimals for now everywhere until we decide to switch to `%` everywhere to be consistent?

Comment (shaundmorris) [reply broker/broker-route-manager/pom.xml]: I'm definitely open to that, I mainly switched these so people at least knew it was a new option.

**Comment by AzGoalie** [comment]
Successfully heroed on winodws 👍

**Comment by shaundmorris** [comment]
build now

File: broker/broker-route-manager/pom.xml (hunk 2 above)
Comment (shaundmorris) [reply broker/broker-route-manager/pom.xml]: I changed it back for consistency purposes

**Pull request merged**